### PR TITLE
Hasura cli

### PR DESCRIPTION
**Pull request by barbaraDorame**
Hasura CLI as a whale 🐋

### CODE REVIEW

**Comment by ivotron** [comment]
can we add this to travis and remove the heroku-cli code? Also, can we add usage information? For examples, you can see this https://github.com/getpopper/library/tree/master/geni

File: hasura-cli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM node:current-slim
+WORKDIR usr/src/app
+ENV NPM_CONFIG_PREFIX /home/node/.npm-global
+ENV PATH $PATH:/home/node/.npm-global/bin
+USER node
+RUN npm install -g hasura-cli
```
Comment (ivotron) [inline hasura-cli/Dockerfile:6]: looks like this is installing the latest (1.2.2) but the image is being tagged as 1.0.0, can you please confirm, and if possible, can you please pin the specific version to this `npm install` command

File: hasura-cli/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Hasura CLI
+
+Docker image for running [Hasura cli commands](https://hasura.io/docs/1.0/graphql/manual/hasura-cli/index.html)
+
+# Example
+-------------------------
```
Comment (ivotron) [inline hasura-cli/README.md:6]: we can remove the horizontal line

**Comment by ivotron** [comment]
@barbaraDorame can you please rebase master? Thanks!

**Comment by ivotron** [comment]
travis prevents variables from being used in forks. So in this case, the push step is making use of `DOCKER_USERNAME` and `DOCKER_PASSWORD`. Can you define them in travis on your end ([see here](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings) for instructions)? It actually doesn't matter what their values are as this step is skipped for branches other than luminous/master

**Comment by barbaraDorame** [comment]
> travis prevents variables from being used in forks. So in this case, the push step is making use of `DOCKER_USERNAME` and `DOCKER_PASSWORD`. Can you define them in travis on your end ([see here](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings) for instructions)? It actually doesn't matter what their values are as this step is skipped for branches other than luminous/master

I did this but it is failling because it's running the last stage anyway

**Comment by ivotron** [comment]
thanks!

**Pull request merged**